dist/angular-patternfly.js

```diff
@@ -308,21 +308,38 @@ angular.module( 'patternfly.card' ).directive('pfAggregateStatusCard', function
 
  <file name="index.html">
    <div ng-controller="ChartCtrl">
-     <div pf-card head-title="Card Title" sub-title="Card Subtitle" filter="filterConfigHeader" style="width: 50%">
-       [Card Contents] <button>Click Me</button> Timeframe filter in header
+     <label class="label-title">Timeframe filter in header</label>
```

Comment:
There is an issue/bug in ng-docs which we haven't gotten to the root of yet.  When I used H's, they get converted to H6's and render strangely.